src/components/commits-page.html

```diff
@@ -1,5 +1,13 @@
 <link rel="import" href="../../bower_components/polymer/polymer.html">
 
+<style type="text/css">
```

Comment:
it's set as none because those messages don't appear unless needed. they get changed to be displayed if you click on the appropriate description or make an error come up. descriptions are at line 130 for example